my-code/src/store/modules/likes/reducer.js

```diff
@@ -0,0 +1,16 @@
+export default function LikesReducer(state = [], action) {
+  switch (action.type) {
+    case 'SET_LIKE':
+      const index = state.indexOf(action.imdbID);
```

Comment:
You could simplify this part using `map`, `filter` and/or `reduce`, it will turn your code easier to read.